06week/checkpoint2.js

```diff
@@ -0,0 +1,107 @@
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+const nums = strNums.map(Number);
+
+// Find the sum of the even values
+const sumEvens = () => {
+    const even = nums.filter(num => num % 2 == 0);
+    const getSum = (total, num) => {
+        return total + num;
+    };
+    const sum = even.reduce(getSum, 0);
+    sumEvens();
```

Comment:
this function calls itself without a stop statement, this would cause an infinite loop